fortanix-sgx-abi/src/lib.rs

```diff
@@ -690,6 +713,15 @@ pub mod async {
         }
     }
 
+    /// Cancel a usercall peviously sent to userspace.
+    #[repr(C)]
+    #[derive(Copy, Clone, Default)]
+    #[cfg_attr(feature = "rustc-dep-of-std", unstable(feature = "sgx_platform", issue = "56975"))]
+    pub struct Cancel {
+        /// Reserved for future use.
+        pub reserved: u64,
```

Comment:
Do you have a use in mind?

Reply:
The timeout idea we discussed in person.